sound/soc/sof/debug.c

```diff
@@ -434,6 +681,21 @@ int snd_sof_dbg_init(struct snd_sof_dev *sdev)
 			return err;
 	}
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_PROBES)
+	err = snd_sof_debugfs_probe_item(sdev, "injection_dma",
+			0444, &injection_dma_fops);
+	if (err < 0)
+		return err;
+	err = snd_sof_debugfs_probe_item(sdev, "probe_points",
+			0644, &ppoints_fops);
+	if (err < 0)
+		return err;
+	err = snd_sof_debugfs_probe_item(sdev, "probe_points_disconnect",
+			0200, &ppoints_discnt_fops);
```

Comment:
can you add a comment on why the permissions are different in all 3 cases? this is far from obvious to me.

Reply:
Addressed in v6.